extensions/big-data-cluster/package.json

```diff
@@ -1,94 +1,93 @@
-            },
-            "mssql-bdc.kubectl-path.mac": {
-              "type": "string",
-              "description": "File path to a kubectl binary."
-            },
-            "mssql-bdc.kubectl-path.linux": {
-              "type": "string",
-              "description": "File path to a kubectl binary."
-            },
-            "mssql-bdc.kubeconfig": {
-              "type": "string",
-              "description": "File path to the kubeconfig file."
-            },
-            "mssql-bdc.knownKubeconfigs": {
-              "type": "array",
-              "description": "File paths to kubeconfig files from which you can select."
-            },
-            "mssql-bdc.outputFormat": {
-              "enum": [
-                "json",
-                "yaml"
-              ],
-              "type": "string",
-              "description": "Output format for Kubernetes specs. One of 'json' or 'yaml' (default)."
-            }
-          },
-          "default": {
-            "mssql-bdc.namespace": "",
-            "mssql-bdc.kubectl-path": "",
-            "mssql-bdc.kubeconfig": "",
-            "mssql-bdc.knownKubeconfigs": []
-          }
-        }
-      }
-    },
-    "commands": [
-      {
-        "command": "mssql.cluster.create",
-        "title": "Create SQL Server big data cluster",
-        "category": "SQL Server"
-      }
-    ]
-  },
-  "dependencies": {
-    "vscode-nls": "^3.2.1",
-    "download": "^6.2.5",
-    "shelljs": "^0.8.3"
-  },
-  "devDependencies": {
-    "mocha-junit-reporter": "^1.17.0",
-    "mocha-multi-reporters": "^1.1.7"
-  }
+	"name": "big-data-cluster",
+	"displayName": "%displayName%",
+	"description": "%description%",
+	"version": "0.0.1",
+	"publisher": "Microsoft",
+	"preview": true,
+	"license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/LICENSE.txt",
+	"icon": "images/sqlserver.png",
+	"engines": {
+		"vscode": "*",
+		"azdata": "*"
+	},
+	"activationEvents": [
+		"*"
+	],
+	"forceReload": true,
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Microsoft/azuredatastudio.git"
+	},
+	"main": "./out/extension",
+	"contributes": {
+		"dataExplorer": {
+			"sqlBigDataCluster": [
+				{
+					"id": "sqlBigDataCluster",
+					"name": "%explorer.title%"
+				}
+			]
+		},
+		"menus": {
+			"view/title": [
+				{
+					"command": "bigDataClusters.command.addController",
+					"when": "view == sqlBigDataCluster",
+					"group": "navigation"
+				}
+			],
+			"view/item/context": [
+				{
+					"command": "bigDataClusters.command.deleteController",
+					"when": "viewItem == bigDataClusters.itemType.ControllerNode",
+					"group": "navigation@1"
+				}
+			]
+		},
+		"configuration": {
+			"type": "object",
+			"title": "%config.title%",
+			"properties": {
+				"bigDataClusterControllers.controllers": {
+					"type": "array"
+				}
+			}
+		},
+		"commands": [
+			{
+				"command": "bigDataClusters.command.addController",
```

Comment:
Should these commands appear in the command palette or not? If they should not, we should add a "commandPalette" menu section and mark all these as `"when": "false"`

Reply:
fixed.